{
    "cve": {
        "data_type": "CVE",
        "data_format": "MITRE",
        "data_version": "4.0",
        "CVE_data_meta": {
            "ID": "CVE-2020-13092",
            "ASSIGNER": "cve@mitre.org"
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": []
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "url": "https://github.com/0FuzzingQ/vuln/blob/master/sklearn%20unserialize.md",
                    "name": "https://github.com/0FuzzingQ/vuln/blob/master/sklearn%20unserialize.md",
                    "refsource": "MISC",
                    "tags": []
                },
                {
                    "url": "https://scikit-learn.org/stable/modules/model_persistence.html#security-maintainability-limitations",
                    "name": "https://scikit-learn.org/stable/modules/model_persistence.html#security-maintainability-limitations",
                    "refsource": "MISC",
                    "tags": []
                }
            ]
        },
        "description": {
            "description_data": [
                {
                    "lang": "en",
                    "value": "** DISPUTED ** scikit-learn (aka sklearn) through 0.23.0 can unserialize and execute commands from an untrusted file that is passed to the joblib.load() function, if __reduce__ makes an os.system call. NOTE: third parties dispute this issue because the joblib.load() function is documented as unsafe and it is the user's responsibility to use the function in a secure manner."
                }
            ]
        }
    },
    "configurations": {
        "CVE_data_version": "4.0",
        "nodes": []
    },
    "impact": {},
    "publishedDate": "2020-05-15T19:15Z",
    "lastModifiedDate": "2020-05-18T12:15Z"
}